### DISCUSSION TITLE
Is there a way to have tasks be submitted without calling `.submit()`

### DISCUSSION THREAD

**Discussion by Andrew-S-Rosen** [PrefectHQ/prefect#10810] [Q&A]
Currently, if you want to submit a task to a `TaskRunner`, you have to call the `.submit()` attribute of the `Task` object. However, this relies the user to modify their underlying Python code, which seems to go against the basic principle behind Prefect. Are there alternate ways to do this?

Current approach:

```python
from prefect import task

@task
def add(a, b):
    return a + b

add.submit(add.submit(1, 2), 3)
```

Note how this is different than the typical `add(add(1, 2), 3)`. When making a `Flow`, this means that the workflow itself will have to be modified to make `.submit()` calls for each `Task`.

I was hoping there might be some way that I could have all tasks automatically submitted to the `TaskRunner` if specified via some kwarg in the `@flow` decorator, but I didn't see such a mechanism.

**Comment by discdiver** [comment]
Interesting idea! Please feel free to open a [feature enhancement issue](https://github.com/PrefectHQ/prefect/issues/new?assignees=&labels=needs%3Atriage%2Cenhancement&projects=&template=3_feature_enhancement.yaml) to share this idea with the team and track it.
